i18n/nuxt/en-US.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Note: these files supposedly should go to https://github.com/modrinth/translations later.